iOS/Controllers/FilesViewController.swift

```diff
@@ -198,3 +161,25 @@ extension FilesViewController {
         }
     }
 }
+
+extension FilesViewController: CoreDataTableViewDataSourceDelegate {
+    typealias Object = File
+    typealias Cell = UITableViewCell
```

Comment:
You don't have to define this explicitly. IT can be inferred from the `configure` method